jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2803,6 +2799,63 @@ YUI.add('juju-env-api', function(Y) {
       }, handler);
     },
 
+    /**
+      Destroy the models with the given tags.
+
+      This method will try to destroy the specified models.
+      It is possible to destroy either other models in the same controller or
+      the current connected model, in which case clients, sooner or later after
+      the server response is received, will likely want to switch to another
+      model not being killed.
+
+      Note that all applications withing the specified models will be destroyed
+      as well, and it's not possible to recover from model's removal.
+      Also note that currently (2016-08-16) nothing prevents this call from
+      destroying the controller model, therefore also disconnecting or even
+      auto-destroying the GUI itself, for instance in the GUI in Juju scenario.
+      For this reason callers are responsible of checking whether a model tag
+      identifies a controller model before calling this method.
+
+      @method destroyModels
+      @param {Array} tags The Juju tags of the models, each one being a string,
+        for instance "model-5bea955d-7a43-47d3-89dd-b02c923e2447".
```

Comment:
I don't recall seeing a model uuid with the prefix "model-" is this a requirement for this method? If so can you expand this comment a bit please.